Install-DVWA.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# Get language prefix
+lang_prefix="${LANG:0:2}"
+
+# Function for verifying the language and displaying the corresponding message
+get_language_message() {
+    if [[ $lang_prefix -eq "es" ]]; then
+        echo -e "$1"
+    else
+        echo -e "$2"
+    fi
+}
+
+# Check if the user is root
+if [ "$EUID" -ne 0 ]; then
+    error_message=$(get_language_message "\e[91mThis script must be run by the root user.\e[0m" "\e[91mEste script debe ejecutarse como usuario root.\e[0m")
+    echo -e "$error_message"
+    exit 1
+fi
+
+# Function to verify the existence of a program
+check_program() {
+    if ! command -v "$1" &>/dev/null; then
```

Comment:
This doesn't work for libraries like php-gd. The better way to check this would be to use `dpkg --get-selections.

```
$ dpkg --get-selections | grep php-gd
php-gd						install
```

Just be careful what you grep for as something that has been removed is flagged as `deinstall`.

There are a few other ways to do it - https://www.baeldung.com/linux/check-how-package-installed